### ISSUE TITLE
When generate outputs, how to get the probility of the outputs? Is there any param to let the model output probility ?

### ISSUE THREAD

**Issue by ShawnALiu** [huggingface/peft#1265]
### Feature request

xx

### Motivation

xx

### Your contribution

xx

**Comment by BenjaminBossan** [comment]
This does not appear to be a PEFT-specific question, so in the future, please ask the question in the appropriate forum.

In general, with transformers language models, when passing `output_scores=True, return_dict_in_generate=True` to `generate`, you should get the logits, which can be used to calculate the probabilities.
